### PR TITLE
Pytest

### PR DESCRIPTION
**Pull request by jssuzanne**
Replace nose test by pytest

### CODE REVIEW

**Comment by petrus-v** [comment]
@jssuzanne this was a first reading I've seen some tiny mistakes in some doc-string that I'll directly commit and add to the branch but I think they were there before that PR. I would like to share with you before doing this 2nd read to make sure where to focus on!

File: anyblok/testing.py [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+    @classmethod
+    def init_configuration_manager(cls, **env):
+        """ Initialise the configuration manager with environ variable
+        to launch the test
+
+        .. warning::
+
+            For the moment we not use the environ variable juste constante
+
+        :param prefix: prefix the database name
```
[the remark this answers is not in the document]
Comment (jssuzanne) [reply anyblok/testing.py]: Oups

File: anyblok/testing.py [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+        cnx.close()
+
+
+@contextmanager
+def tmp_configuration(**values):
+    """Add Configuration value only in the contextmanager
+    ::
+
+        with TestCase.Configuration(db_name='a db name'):
+            self.assertEqual(Configuration.get('db_name'), 'a db name')
```
[the remark this answers is not in the document]
Comment (jssuzanne) [reply anyblok/testing.py]: oups, done in another branch

File: anyblok/testing.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jssuzanne) [reply anyblok/testing.py]: done

File: anyblok/conftest.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+@pytest.fixture(scope='session')
+def init_session(request):
+    # Init registry
+    additional_setting = {'unittest': True}
+
+    if len(BlokManager.list()) == 0:
+        BlokManager.load()
+    registry = RegistryManager.get(Configuration.get('db_name'),
+                                   **additional_setting)
+    registry.commit()
```
[the remark this answers is not in the document]
Comment (jssuzanne) [reply anyblok/conftest.py]: To be sure that the table exist and they are not destroy by a hard rollback

**Pull request merged**